packages/generator/generators/draft/templates/README.txt

```diff
@@ -0,0 +1,27 @@
+---
```

Comment:
@didoesdigital Not sure what a sensible default for this file should be. I'm hoping this will prompt the contributor to reach out to a designer for advice.

Reply:
@ActuallyACat Nice one. It's a good prompt.

Change `README.txt` to `README.md`?

Reply:
This is just a template file - it will be copied, have the variables injected into it, and spit out as an `mdx` file: https://github.com/cultureamp/kaizen-design-system/pull/720/files#diff-53cc4ef3807f9e09838ac70f0218da41R49-R50